src/components/icon/assets/thumb_down.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
 <path d="M20.4 4H17.4L15.4 2H5.6L2 5.6V14.4L4.6 17H9V20.4L10.6 22H14.4L16 20.4V17.4L17.4 16H22V5.6L20.4 4ZM16 14.6L14 16.6V19.6L13.6 20H11.4L11 19.6V15H5.4L4 13.6V12H9V11H4V8H9V7H4V6.4L6.4 4H14.6L16 5.4V14.6ZM20 14H18V6H19.6L20 6.4V14Z" fill="white"/>
```

Comment:
I usually remove the path's fill color as well. Not sure if it's needed to be removed.